solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/LTRQParserPlugin.java

```diff
@@ -50,7 +51,20 @@
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
   @Override
-  public void init(@SuppressWarnings("rawtypes") NamedList args) {}
+  public void init(@SuppressWarnings("rawtypes") NamedList args) {
+    LTRThreadInterface.maxThreads  = getInt(args.get("LTRMaxThreads"), LTRThreadInterface.DEFAULT_MAX_THREADS, "LTRMaxThreads");
+    LTRThreadInterface.maxQueryThreads = getInt(args.get("LTRMaxQueryThreads"), LTRThreadInterface.DEFAULT_MAX_QUERYTHREADS, "LTRMaxQueryThreads");
+  }
+  private int getInt(Object thObj, int defValue, String paramName){
+     if (thObj != null) {
+       try{
+          return Integer.parseInt(thObj.toString());
+       }catch(NumberFormatException nfe){
+          log.error("{}: {} not an integer using defaultValue: {}", nfe.toString(), paramName, defValue);
```

Comment:
Should we really be silently failing here?  I feel like we should still throw the exception, just like if you specified a garbage class in the schema, startup will/should fail.